CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 /src/handbook/development @knolleary
 /src/handbook/marketing @iskerrett
 /src/handbook/product @MarianRaphael
+/src/handbook/sales @zackwasly
```

Comment:
@knolleary How do I give Zack merge access, do you happen to know? I really did try in GH, but it's not obvious to me

Reply:
Everything I can see says Zack has write access to all of our repos - https://github.com/orgs/FlowFuse/people/zackwasli

What is the symptom you're seeing here?